.github/workflows/nightly.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         run: echo "today=$(date '+%Y%m%d')" >> $GITHUB_OUTPUT
 
       - name: Release
-        uses: softprops/action-gh-release@1e07f4398721186383de40550babbdf2b84acfc5 # v0.1.14
+        uses: softprops/action-gh-release@ 4634c16e79c963813287e889244c50009e7f0981 # https://github.com/softprops/action-gh-release/pull/406
```

Comment:
There is no version? Is clearer to put a version and not a commit.

Reply:
It seems it has not been published as version. Maybe we can wait?

Reply:
@McGiverGim It's has been released now :tada: